lib/l10n/app_en.arb

```diff
@@ -951,5 +954,291 @@
+    "placeholders": {
+      "libraryName": {
+        "type": "String"
+      }
+    }
+  },
+  "onlyShowFullyDownloaded": "Only show fully downloaded albums",
+  "@onlyShowFullyDownloaded": {
+    "description": "Tooltip for button that hides all partially downloaded albums"
+  },
+  "filesystemFull": "Remaining downloads cannot be completed.  The filesystem is full.",
+  "@filesystemFull": {
+    "description": "Message shown when a download cannot complete due to the filesystem being full."
+  },
+  "connectionInterrupted": "Connection interrupted, pausing downloads.",
+  "@connectionInterrupted": {
+    "description": "Message shown when the connection is interrupted during a download."
+  },
+  "connectionInterruptedBackground": "Connection was interrupted while downloading in the background.  This can be caused by OS settings.",
+  "@connectionInterruptedBackground": {
+    "description": "Message shown when the connection is interrupted during a download while in the background.  It advises checking OS settings, as they may cause background downloads to be blocked."
+  },
+  "connectionInterruptedBackgroundAndroid": "Connection was interrupted while downloading in the background.  This can be caused by enabling 'Enter Low-Priority State on Pause' or OS settings.",
+  "@connectionInterruptedBackgroundAndroid": {
+    "description": "Message shown when the connection is interrupted during a download while in the background on android.  It advises checking OS settings, as they may cause background downloads to be blocked, or checking the  'Enter Low-Priority State on Pause' setting."
+  },
+  "activeDownloadSize": "Downloading...",
+  "@activeDownloadSize": {
+    "description": "Message shown for download size while item is still being downloaded"
+  },
+  "missingDownloadSize": "Deleting...",
+  "@missingDownloadSize": {
+    "description": "Message shown for download size while item is still being downloaded"
+  },
+  "syncingDownloadSize": "Syncing...",
+  "@syncingDownloadSize": {
+    "description": "Message shown for download size while item is still syncing."
+  },
+  "runRepairWarning": "The server could not be contacted to finalize downloads migration.  Please run 'Repair Downloads' from the downloads screen as soon as you are back online.",
+  "@runRepairWarning": {
+    "description": "Message shown when an error occurs during the downloads repair step of the Isar downloads migration, which requires a server connection."
+  },
+  "downloadSettings": "Download Settings",
+  "@downloadSettings": {},
+  "showPlaylistSongs": "Show Items with Unknown Library.",
+  "@showPlaylistSongs": {
+    "description": "Title of setting that shows songs/albums from an unknown library in all libraries when on, or in no libraries when off."
+  },
+  "showPlaylistSongsSubtitle": "Some items may be downloaded with an unknown library.  They can show in all libraries, or none.",
```

Comment:
again here, "Some items may not be tied to a specific library, like some playlists. Should they always be shown, no matter which library is selected?"

Reply:
Can you change the second sentence to something like "Turn off to only show these in their original collection."  
And also, please replace "item" with "media", "media files" or "files". Items is too technical for my taste :)

Reply:
Does 'Some media files may be downloaded with an unknown library.  Turn off to hide these outside their original collection.' seem good?

Reply:
Yes that seems nice. If this could also apply to stuff like albums (e.g. without an artist?), then I'd use just media instead of media files, but just go ahead and change it, then we're done 😁